.github/workflows/test-scripts.yml

```
@@ -0,0 +1,48 @@
+name: "Cross platform script test"
+
+on:
+  push:
+    branches:
+      - master
+      - develop
+      - 'feature/**'
+  pull_request:
+
+jobs:
+  test-scripts:
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-20.04, windows-2022]
+    runs-on: ${{ matrix.os }}
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+
+      - name: Set up Java
+        uses: actions/setup-java@v3
```

Blocking: Can you pin the dependency to the full commit hash ID and not tags? Thanks.

```suggestion
        uses: actions/setup-java@b36c23c0d998641eff861008f374ee103c25ac73
```

---

updated